pallets/pool-system/src/impls.rs

```diff
@@ -454,12 +454,20 @@ impl<T: Config> ChangeGuard for Pallet<T> {
 			}
 		}
 
-		allowed
+		let change = allowed
 			.then(|| {
 				NotedChange::<T>::remove(pool_id, change_id);
 				change
 			})
-			.ok_or(Error::<T>::ChangeNotReady.into())
+			.ok_or(Error::<T>::ChangeNotReady.into())?;
```

Comment:
This is already notified in the consumer side of `released()`: https://github.com/centrifuge/centrifuge-chain/blob/main/pallets/oracle-collection/src/lib.rs#L244

Not sure if we want another event here.

Reply:
AFAICS, we are not emitting such an event for pool fees for loans. IMO, it makes sense to add this here.

Reply:
Also the Events are quite confusing without showing that this came from a change that was released: 
* see: https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffullnode.demo.k-f.dev#/explorer/query/0xdcd2cf1b78343a46ffea6b4a959e7644ae59aca614336a7648a6423c53dd6ca1

And if we are noting a change we are also emitting an event. I would keep it.